test/refract/data/test-Select.cc

```diff
@@ -0,0 +1,15 @@
+//
+//  test/refract/data/test-Select.cc
+//  test-librefract
+//
+//  Created by Thomas Jandecka on 27/08/2017
+//  Copyright (c) 2017 Apiary Inc. All rights reserved.
+//
+
+#include "catch.hpp"
+
+#include "refract/data/Select.h"
+
+using namespace refract;
+using namespace data;
+
```

Comment:
Missing test implementation.
Same for:
- ref
- option
- object
- null
- holder
- enum